### DISCUSSION TITLE
How to choose your maintenance window for Teleport Cloud

### DISCUSSION THREAD

**Discussion by WilliamLoy** [gravitational/teleport#23141] [General]
You can specify the time of day you'd prefer your Teleport Cloud tenant to be upgraded. This allows admins to plan around upgrades and communicate timeframes to end-users.

1. Log into your Teleport Cloud Web Interface. 
2. Click your username in the upper right-hand corner to reveal the drop-down menu. 
3. Click *Help & Support*
![upgrade](https://user-images.githubusercontent.com/18149544/225418150-2e068460-dfe2-40ec-a694-ff031cf05f68.jpg)
4. Scroll to  the bottom of the screen to the section titled *Scheduled Upgrades*
![schedule](https://user-images.githubusercontent.com/18149544/225417972-174297b4-6bbc-4c19-ab24-154bf77b75cd.jpg)
5. Click the edit icon on the right of the *Window Start Time:* line. 
6. Select your desired maintenance window time. 
![time](https://user-images.githubusercontent.com/18149544/225418308-53779417-6a3f-4949-86dc-839ef9c06401.jpg)

**Comment by cmeury** [comment]
Since we can only specify the start time, is there a maximum window size?

**Comment by milos-teleport** [comment]
This information is now available here:
https://goteleport.com/docs/upgrading/cloud-cluster-updates/#maintenance-windows
